Sidebar, Dashboard, and My Projects Update

ezramoss opened the pull request:
This addresses issue #11 and #19 . Updated Dashboard and My Projects Page. Adjusted Sidebar active tab width.

ezramoss committed:
Opportunities View Changes

In Opportunities view, changed header, added tabs. Added filter and sort icon buttons. Added search bar. Changed card button text and color. Changed sidebar color of active tab.

ezramoss committed:
Refactored Dashboard Views & Minor CSS Changes

Removed repeat code/CSS, Removed App.tsx/css code, moved some css to index.css. fixed background color display error on opportunities tab.  added scroll bar to project cards. refactored some cards to display via mappings.

ezramoss committed:
Added Inter Font, minor padding changes.

ezramoss committed:
Initial Profile View

Initial Profile View

ezramoss committed:
Updated Profile Page

Refactored .TSX and .CSS to be more readable, edited experience font, added total yoe.

ezramoss committed:
Visual Enhancements

- Edited Profile Page to reduce gap at role listing
- Edited Opportunities page to make card/tabs text and colors more inline with Figma Design

ezramoss committed:
Updated Opportunities Page / Card Component

Changed font colors, font sizes, font weights, background colors, rounded corners, filter icon, and scrollbar gap to be more inline with Figma design.

ezramoss committed:
Sidebar fixes

Updated sidebar with new Figma design. Updated scroll bar on opportunities page. Created basic settings and tools page.

ezramoss committed:
Sidebar, Dashboard, and My Projects Update

Updated sidebar active tab width. Updated Dashboard and My Projects Page UI in relation to Figma design.

ezramoss commented:
@AfnanZ Can you please review my changes and let me know your thoughts? Thank you.

Jbutler-helpful commented:
@AfnanZ have you had a chance to review this Pull Request?

AfnanZ commented:
All cool @ezramoss, except one small thing we have been missing.
<img width="240" height="67" alt="Screenshot 2025-09-04 at 12 03 47 AM" src="https://github.com/user-attachments/assets/300075ab-b742-473d-a2fc-36dd659ce36c" />

However, updated design looks like this:
<img width="240" height="39" alt="Screenshot 2025-09-04 at 12 04 54 AM" src="https://github.com/user-attachments/assets/34549988-b6f9-41b9-bc94-07a60dd5cdef" />

Is this change possible?

Thank you so much for your patience :)

Jbutler-helpful commented:
@ezramoss let me know when resolved and ready for review by Peter

Jbutler-helpful commented:
@gurupreeth6 would do a review of this pr please

ezramoss committed:
Dashboard Changes

- Changed Get Started Button
- Changed Sort by drop down
- Changed Website Title and Icon
